pkg/processor/trigger/types.go

```diff
@@ -47,7 +47,7 @@ type AnnotationConfigField struct {
 }
 
 type Configuration struct {
-	functionconfig.Trigger
+	*functionconfig.Trigger
```

Comment:
why is this needed?

Reply:
because below there is a method where we firstly did init of configuration object

```	
configuration := &Configuration{
		Trigger:              triggerConfiguration,
		RuntimeConfiguration: runtimeConfiguration,
		ID:                   id,
	}
```

and then we tried to change 

```
	if triggerConfiguration.WorkerTerminationTimeout == "" {
		triggerConfiguration.WorkerTerminationTimeout = functionconfig.DefaultWorkerTerminationTimeout
	}
	...
	runtimeConfiguration.WorkerTerminationTimeout = workerTerminationTimeout
```

Which caused the bug where we didn't really change anything